pipelines/batch/src/main/java/org/openmrs/analytics/FetchSearchPageFn.java

```diff
@@ -95,13 +141,19 @@
 	}
 	
 	@Setup
-	public void setup() {
+	public void setup() throws SQLException, PropertyVetoException {
 		FhirContext fhirContext = FhirContext.forR4();
+		fhirContext.getRestfulClientFactory().setSocketTimeout(20000);
 		fhirStoreUtil = FhirStoreUtil.createFhirStoreUtil(sinkPath, sinkUsername, sinkPassword,
 		    fhirContext.getRestfulClientFactory());
 		openmrsUtil = new OpenmrsUtil(sourceUrl, sourceUser, sourcePw, fhirContext);
 		fhirSearchUtil = new FhirSearchUtil(openmrsUtil);
 		parquetUtil = new ParquetUtil(parquetFile, secondsToFlush, rowGroupSize, stageIdentifier + "_");
+		if (!sinkDbUrl.isEmpty()) {
+			jdbcWriter = new JdbcResourceWriter(getJdbcConnectionUtil(jdbcDriverClass, sinkDbUrl, sinkDbUsername,
```

Comment:
extract `getJdbcConnectionUtil(jdbcDriverClass, sinkDbPassword, initialPoolSize, maxPoolSize).getDataSource())` to a separate variable and pass that variable